gradle/anko-asynchronous-tasks/MyApplication.iml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`MyApplication.iml` is unlikely to be useful without the corresponding `.idea` directory.
You can just drop as it easy to import the existing Gradle project to IntelliJ IDEA.